plugins/instructions/SOS/README.md

```
@@ -0,0 +1,21 @@
+
+1. Open (https://sos-auth.vercel.app/)[https://sos-auth.vercel.app/]
+2. Follow directions by inputing your emergenecy contacts
+
+
+## Troubleshooting
+
+[Common issues and solutions coming soon]
+
+---
+
+> Experimental feature. Feedback: shreybirmiwal@gmail.com
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)